scripts/generate-stocks-csv.js

```
@@ -0,0 +1,301 @@
+  FOREST_TYPES.forEach((ft) => {
+    h.push(`${ft}_surface_ha`)
+    h.push(`${ft}_sol_stock_de_reference_tC_ha-1`)
+    h.push(`${ft}_biomasse_vivante_stock_de_reference_tC_ha-1`)
+    h.push(`${ft}_biomasse_morte_stock_de_reference_tC_ha-1`)
+    h.push(`${ft}_litiere_stock_de_reference_tC_ha-1`)
+    h.push(`${ft}_stock_total_tC`)
+  })
+  // Haies
+  h.push('haies_biomasse_stock_de_reference_tC_ha-1')
+  h.push('haies_stock_total_tC')
+  HAIES_GT_KEYS.forEach(([, col]) => h.push(col))
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)